### PR TITLE
[DomCrawler] Add hint about Form::getName method

### PR DESCRIPTION
**Pull request by JustBlackBird**
This PR describes some use cases of `Symfony\Component\DomCrawler\Form::getName` method

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -801,6 +801,12 @@ their type::
     $form['my_form[field][O]']->upload('/path/to/lucas.jpg');
     $form['my_form[field][1]']->upload('/path/to/lisa.jpg');
 
+.. tip::
+
+    All field names in forms generated by symfony are prefixed with from's name.
```
Comment (OskarStark) [inline testing.rst]: from‘s -> form‘s

Comment (JustBlackBird) [reply testing.rst]: Fixed

File: testing.rst [hunk 2]
```diff
@@ -801,6 +801,12 @@ their type::
     $form['my_form[field][O]']->upload('/path/to/lucas.jpg');
     $form['my_form[field][1]']->upload('/path/to/lisa.jpg');
 
+.. tip::
+
+    All field names in forms generated by symfony are prefixed with form's name.
```
Comment (OskarStark) [inline testing.rst]:
```suggestion
    All field names in forms generated by Symfony are prefixed with the form's name.
```

Comment (JustBlackBird) [reply testing.rst]: @OskarStark fixed

File: testing.rst [hunk 3]
```diff
@@ -801,6 +801,12 @@ their type::
     $form['my_form[field][O]']->upload('/path/to/lucas.jpg');
     $form['my_form[field][1]']->upload('/path/to/lisa.jpg');
 
+.. tip::
+
+    All field names in forms generated by Symfony are prefixed with the form's name.
+    If the name is autogenerated you can get it using
```
Comment (xabbuh) [inline testing.rst]: This sounds misleading IMO. For the method it doesn't really matter how the name originates. Maybe we should reword it to something like this:

> Use the `getName()` method if you need to access the name of a form.

Comment (JustBlackBird) [reply testing.rst]: Name of the field is not that obvious on the first sight, so I believe a hint is needed to make it clear. Also, using `getName` method in tests can make them a little less fragile.

**Comment by javiereguiluz** [comment]
Dmitriy, thanks a lot for contributing this nice little feature and thanks for providing the docs too! We merged it with some very minor rewords as proposed by @xabbuh.

Congrats on your first Symfony Docs contribution!
